tests/tests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.1;netcoreapp3.1;netcoreapp5.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1;netcoreapp5.0</TargetFrameworks>

```

Comment:
there is no such thing as `netcoreapp5.0`, it should be `net5.0` and give .NET 6 is also out, you can potentially add `net6.0`